bokehjs/gulp/tasks/generate_defaults.py

```diff
@@ -104,8 +104,5 @@ def output_defaults_module(filename, defaults):
 
 output_defaults_module(filename = os.path.join(dest_dir, 'common/defaults.coffee'),
                        defaults = all_json)
-output_defaults_module(filename = os.path.join(dest_dir, 'widget/defaults.coffee'),
+output_defaults_module(filename = os.path.join(dest_dir, 'models/widgets/defaults.coffee'),
```

Comment:
I agree - the locations weren't great initially and the new proposed locations aren't any better. I'd prefer to update their locations in a separate PR. I also want to rename the modules that register all of the models:

`common/models.coffee` (for models)
`models/widgets/main.coffee` (for widgets)